unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/HUD/BackpackEditorHUDV2/BackpackEditorHUDController.cs

```diff
@@ -243,9 +243,10 @@ async UniTaskVoid LoadUserProfileAsync(UserProfile userProfile, CancellationToke
                             }
                         }
 
-                        EquipWearable(wearable, EquipWearableSource.None, false, false);
+                        EquipWearable(wearable, EquipWearableSource.None, false, false, false);
                     }
 
+                    avatarSlotsHUDController.Recalculate(model.forceRender);
```

Comment:
Make sure nothing happens if we pass in a null value to the `Recalculate()` method.